.vscode/settings.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
I think it would be better to gitignore this file. As in most cases it's used only for personal preferences and not related to the project itself.